### PR TITLE
Set severity when sending notification

### PR DESCRIPTION
**Pull request by gabriel-farache**
Closes https://issues.redhat.com/browse/FLPATH-1553
Set severity when sending notification

### CODE REVIEW

**Review by ydayagi — APPROVED** [review]

**Review by masayag — APPROVED** [review]

File: .github/workflows/mta-v7.x-e2e.yaml [hunk 1]
```diff
@@ -115,18 +115,18 @@ jobs:
...
           # Set the endpoint to the tackle-ui service
-          yq --inplace '.spec.podTemplate.container.env |= ( . + [{"name": "QUARKUS_REST_CLIENT_MTA_JSON_URL", "value": "http://tackle-ui.my-konveyor-operator.svc:8080"}, {"name": "BACKSTAGE_NOTIFICATIONS_URL", "value": "http://orchestrator-backstage.default.svc.cluster.local:7007/"}] )' manifests/01-sonataflow_mta-analysis-v7.yaml
+          yq --inplace '.spec.podTemplate.container.env |= ( . + [{"name": "QUARKUS_REST_CLIENT_MTA_JSON_URL", "value": "http://tackle-ui.my-konveyor-operator.svc:8080"}, {"name": "BACKSTAGE_NOTIFICATIONS_URL", "value": "http://orchestrator-backstage.default.svc.cluster.local:7007/"}] )' manifests/04-sonataflow_mta-analysis-v7.yaml
           
           # Disable persistence for e2e tests
-          yq e '.spec.persistence = {}' -i manifests/01-sonataflow_mta-analysis-v7.yaml
+          yq e '.spec.persistence = {}' -i manifests/04-sonataflow_mta-analysis-v7.yaml
           sed -i '/quarkus\.flyway\.migrate-at-start=true/d' manifests/03-configmap_mta-analysis-v7-props.yaml
 
           echo "manifests/03-configmap_mta-analysis-v7-props.yaml"
```
Comment (masayag) [inline .github/workflows/mta-v7.x-e2e.yaml:124]: pls change this to `01-...`

Comment (masayag) [reply .github/workflows/mta-v7.x-e2e.yaml]: sorry, did that already

**Review by masayag — APPROVED** [review]

File: .github/workflows/mta-v7.x-e2e.yaml (hunk 1 above)
Comment (masayag) [reply .github/workflows/mta-v7.x-e2e.yaml]: test still fails, probably misses the patching of the secret. not going to do it now. merging.

Comment (gabriel-farache) [reply .github/workflows/mta-v7.x-e2e.yaml]: Not sure about that, the MTA V7 e2e tests are failing for quite some time I think https://github.com/parodos-dev/serverless-workflows/actions/workflows/mta-v7.x-e2e.yaml

Comment (masayag) [reply .github/workflows/mta-v7.x-e2e.yaml]: so probably more work is required. one day...

**Pull request merged**